org.eclipse.jdt.core/model/org/eclipse/jdt/core/compiler/CompilationParticipant.java

```diff
@@ -152,6 +152,37 @@ public void processAnnotations(BuildContext[] files) {
 	// do nothing by default
 }
 
+/**
+ * Provisional API for experimental support for an alternative compiler
+ * and can be removed without notice.
```

Comment:
this looks like a case for `@noreference`

Reply:
nice. @noreference tag added. thanks.